src/main/java/org/jspecify/experimentaldonotuse/DefaultNullable.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2018-present The Code Analysis Annotations Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jspecify.experimentaldonotuse;
+
+import static java.lang.annotation.ElementType.PACKAGE;
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.CLASS;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Documented
+@Target({TYPE, PACKAGE})
+@Retention(CLASS)
+public @interface DefaultNullable {}
```

Comment:
For at least our first release, we'll have only `@DefaultNonNull` and `@Nullable`. You can remove the others now, or we can do it in a followup.